package.json

```diff
@@ -18,5 +18,9 @@
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": false,
     "step_definitions": "cypress/integration/step_definitions/Ui"
+  },
+  "dependencies": {
+    "xmldom": "^0.6.0",
+    "xpath": "^0.0.33"
```

Comment:
Where did you use these dependencies? If you haven't used them, can you delete from here and do `npm install` again? Then you can recommit

Reply:
done